.github/workflows/ci.yml

```diff
@@ -0,0 +1,77 @@
+name: CI
+
+on:
+  push:
+  pull_request:
+  schedule:
+    - cron:  '0 2 * * *'
+
+jobs:
+  test:
+    strategy:
+      matrix:
+        rust: ["stable", "beta", "nightly"]
+        os: [ubuntu-18.04, macos-latest]
+    name: Cargo test
+    runs-on: ${{ matrix.os }}
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v1
+    - name: Install toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: ${{ matrix.rust }}
+        override: true
+    - name: Execute cargo test
+      run: cargo test --all --tests --examples
+  wasm_test:
+    name: Cargo test for wasm
+    runs-on: ubuntu-18.04
+    steps:
+    - name: Checkout sources
+      uses: actions/checkout@v1
+    - name: Install toolchain
+      uses: actions-rs/toolchain@v1
+      with:
+        profile: minimal
+        toolchain: stable
+        override: true
+    - name: Setup wasm-bindgen
+      run: |
+        rustup target add wasm32-unknown-unknown &&
+        sudo apt update && sudo apt install -y firefox-geckodriver &&
+        wget https://github.com/rustwasm/wasm-bindgen/releases/download/0.2.54/wasm-bindgen-0.2.54-x86_64-unknown-linux-musl.tar.gz &&
```

Comment:
We should add a note to the relevant `Cargo.toml` that when updating `wasm-bindgen` in the main crate, CI needs updated as well.